features/admin_center_manage_display.feature

```diff
@@ -452,33 +452,23 @@ Feature: Manage Display
     And the "Status" field should contain "Inactive"
     And I should see "The alert for this site has been updated."
 
-  Scenario: Editing the Visual Design Settings when "Show Redesign Display Settings" is false
-    Given the following Affiliates exist:
-      | display_name | name       | contact_email   | first_name | last_name |
-      | agency site  | agency.gov | john@agency.gov | John       | Bar       |
-    And I am logged in with email "john@agency.gov"
-    When I go to the agency.gov's Visual Design page
-    Then I should see "Visual design (new)"
-    Then I should see a link to "Preview Redesigned Display" with url that ends with "/search?affiliate=agency.gov&redesign=true"
-    And the page body should not contain "These settings are for preview purposes only."
-
   @javascript
-  Scenario: Editing the Visual Design Settings when "Show Redesign Display Settings" is true
+  Scenario: Editing the Visual Design Settings when "Use Redesigned Results Page" is true
     Given the following Affiliates exist:
-      | display_name | name       | contact_email   | first_name | last_name | show_redesign_display_settings |
-      | bing site    | agency.gov | john@agency.gov | John       | Bar       | true                           |
+      | display_name | name       | contact_email   | first_name | last_name | use_redesigned_results_page |
+      | bing site    | agency.gov | john@agency.gov | John       | Bar       | true                        |
     And I am logged in with email "john@agency.gov"
     When I go to the agency.gov's Visual Design page
     Then I should see "Visual design (new)"
-    And the page body should contain "These settings are for preview purposes only."
+    And the page body should not contain "These settings are for preview purposes only."
     And I should see "Fonts & Colors" within the navigation tabs
     And I should not see "Results Format" within the navigation tabs
     And I should see "Image Assets" within the navigation tabs
     And I should see "Header & Footer" within the navigation tabs
 
     Given the following SearchGov Affiliates exist:
-      | display_name    | name       | contact_email   | first_name | last_name | show_redesign_display_settings |
-      | searchgov site  | agency.gov | john@agency.gov | John       | Bar       | true                           |
+      | display_name    | name       | contact_email   | first_name | last_name | use_redesigned_results_page |
```

Comment:
This test setup doesn't match the conditions given in the Scenario description.  I'd break this into two scenarios - one where use_redesigned_results_page is true, and the other where it is false.  For the false condition, you'll only really need to test for the presence of the preview warning message, and the bulk of the tests should be kept under a true condition scenario since, in the future, all sites will be true and that preview message will be permanently removed.